src/platforms/gbm-kms/server/kms/real_kms_output_container.cpp

```diff
@@ -14,10 +14,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#define MIR_LOG_COMPONENT "gbm-kms"
```

Comment:
There should be no need for this: `MIR_LOG_COMPONENT_FALLBACK="gbm-kms"` is in CMakeLists.txt